### ISSUE TITLE
Dist folder missing on 19.1.1

### ISSUE THREAD

**Issue by rubenccreis** [ag-grid/ag-grid#2752]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of ag-Grid'
[] support request => see 'Requesting Community Support'
```

**Current behavior**
Hi guys.
I have a problem when I run the command **bower install ag-grid-community#^19.1.1**.
I can't found dist folder with the 4 bundles.

**Expected behavior**
Like the previous version, I think that dist folder is missing on root folder.

**Please tell us about your environment:**
Node v0.10.33
Bower v1.8.4
IDE: Visual Studio Code 

* **ag-Grid version:** 19.1.1

* **Browser:** NA

* **Language:** NA

**Comment by makinggoodsoftware** [comment]
Hi,

Your ticket has been flagged as in_zendesk, it means that we recognize this as an issue worth investigating so we have moved it to our official support channel for customers (zendesk)

We will carry on there with the investigation and we will update you as soon as possible.

Thanks

**Comment by makinggoodsoftware** [comment]
Hi,

Just to confirm that this should have been fixed in 19.1.2

Thanks
